avaframe/in3Utils/cfgUtils.py

```diff
@@ -479,3 +479,47 @@ def filterSims(avalancheDir, parametersDict, specDir=''):
     simNameList = simDF['simName'].tolist()
 
     return simNameList
+
+def orderSimFiles(avalancheDir, inputDir, varParList, ascendingOrder, specDir=''):
+    """ Filter simulations results using a list of parameters and a flag if in ascending or descending order
+
+        Parameters
+        -----------
+        avalancheDir: str
+            path to avalanche directory
+        inputDir: str
+            path to simulation results
+        varParList: str or list
+            simulation configuration parameters for ordering simulations
+        ascendingOrder: bool
+            True if simulations shall be ordered in ascending order regarding varPar
+        specDir: str
+            path to a directory where simulation configuration files can be found - optional
+
+        Returns
+        --------
+        dataDF: pandas dataFrame
+            dataFrame of simulation results (fileName, ... and values for parameters in varParList)
+    """
+
+
+    # load dataFrame for all configurations
```

Comment:
what if the data is not coming from com1DFA: no ini files

Reply:
I added an if comModule='com1DFA' do ordering otherwise fetch simulation results from folder without ordering - but this should probably be done differently that we check for configuration files also from other modules and if there are none then..